src/components/mgt-person/mgt-person.ts

```diff
@@ -324,9 +341,154 @@ export class MgtPerson extends MgtTemplatedComponent {
       return this.renderNoData();
     }
 
+    return html`
+      ${imageHtml}
+    `;
+  }
+
+  /**
+   * Render presence for the person.
+   *
+   * @protected
+   * @param
+   * @memberof MgtPersonCard
+   */
+  protected renderPresence(presence?: Presence): TemplateResult {
+    if (!this.showPresence) {
+      return html``;
+    }
+
+    presence = presence || this.personPresence;
+    if (!presence) {
+      return html``;
+    }
+
+    let statusClass = null;
+    // attach appropriate css class to show different icons
+    switch (presence.availability) {
+      case 'DoNotDisturb':
+        statusClass = 'presence-dnd';
+        break;
+      case 'BeRightBack':
+        statusClass = 'presence-away';
+        break;
+      case 'Available':
+        switch (presence.activity) {
+          case 'Available':
+            statusClass = 'presence-available';
+            break;
+          case 'OutOfOffice':
+            statusClass = 'presence-oof-available';
+            break;
+        }
+        break;
+      case 'Busy':
+        switch (presence.activity) {
+          case 'OutOfOffice':
+            statusClass = 'presence-oof-busy';
+            break;
+          default:
+            // 'Busy', 'InACall', 'InAMeeting'
+            statusClass = 'presence-busy';
+            break;
+        }
+        break;
+      case 'Away':
+        switch (presence.activity) {
+          case 'Away':
+            statusClass = 'presence-away';
+            break;
+          case 'OutOfOffice':
+            statusClass = 'presence-oof-offline';
+            break;
+        }
+        break;
+      case 'Offline':
+        switch (presence.activity) {
+          case 'Offline':
+            statusClass = 'presence-offline';
+            break;
+          case 'OutOfOffice':
+            statusClass = 'presence-oof-offline';
+            break;
+        }
+        break;
+      default:
+        statusClass = 'presence-offline';
+        break;
+    }
+
+    const presenceClasses = {
+      'ms-Icon': true,
+      'presence-basic': true
+    };
+
+    presenceClasses[statusClass] = true;
+    // workaround because SkypeArrow icon from fluent doesn't work ¯\_(ツ)_/¯
+    let iconHtml = null;
+    if (statusClass === 'presence-oof-offline') {
+      iconHtml = html`
+        <div class="ms-Icon presence-basic presence-oof-offline-wrapper">
+          <svg class="presence-oof-offline" viewBox="0 0 12 10" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
You can move this with all the other svgs: https://github.com/microsoftgraph/microsoft-graph-toolkit/blob/master/src/utils/SvgHelper.ts

Reply:
hmmm can you add class to it?